troposphere/static/css/app/quota.scss

```diff
@@ -0,0 +1,40 @@
+textarea, select{
```

Comment:
We need to scope this selector, otherwise it will effect every textarea and select field in the entire application, not just on your Quota page.